x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email/email_connector.tsx

```diff
@@ -56,7 +56,7 @@ export const EmailActionConnectorFields: React.FunctionComponent<
             )}
             helpText={
               <EuiLink
-                href={`${docLinks.ELASTIC_WEBSITE_URL}guide/en/kibana/${docLinks.DOC_LINK_VERSION}/email-action-type.html#configuring-email`}
+                href={`${docLinks.links.alerting.emailAction}#configuring-email`}
```

Comment:
> href={`${docLinks.links.alerting.emailAction}#configuring-email`}

I think the downside of putting the section anchor here instead of adding a new keyword in the doc link service, is that if the section is removed or changed, our link checking won't catch it. I think in cases where the section no longer exists, the link will default to the top of the page. If that's acceptable, then this is an okay implementation.

Reply:
I will change it to have an own link - I think it's better to maintain 👍